src/ray/raylet/node_manager.cc

```diff
@@ -466,7 +466,7 @@ void NodeManager::HeartbeatAdded(const ClientID &client_id,
     if (state != TaskState::INFEASIBLE) {
       // Don't unsubscribe for infeasible tasks because we never subscribed in
       // the first place.
-      RAY_CHECK(task_dependency_manager_.UnsubscribeDependencies(task_id));
+      RAY_CHECK(task_dependency_manager_.UnsubscribeAllDependencies(task_id));
```

Comment:
I think we probably want `UnsubscribeDependencies` here. This code is for tasks that were queued, so we were trying to `ray.get` their arguments, but we no longer need to. Since the tasks haven't started running yet, they shouldn't have made any `ray.wait` calls.